mote/static/css/mote.css

```diff
@@ -274,3 +274,9 @@ a {
 .latest-meetings-list li {
     list-style-type: none;
 }
+
+.select2-results__option--highlighted[aria-selected] {
+    background-color: #294172 !important;
+    color: white;
+  }
+  
```

Comment:
Yes, that's exactly what I was conveying. Making changes to this custom-made CSS file would be much better than adding those changes to an imported one for who knows Select2 might be used somewhere else and not need those theming options.

If the consequence of these changes look the same as before, I think it is ready for a merge.

Reply:
This is giving out the same output!! Thanks for reviewing!